### PR TITLE
[MODULAR] Biodome Un-Soulening

### PR DESCRIPTION
**Pull request by TripleShades**
## About The Pull Request
Redecorates Biodome from being an eyesore.
Also fixes various things, such as map parity (equipment/objects/decals, etc, that is expected across all maps per area), and reported issues from the issues channel/biodome issues thread.
Also also adds in a Blacksmith workshop, NT Rep/Consult office, and a Barber/Salon.

Please see changelog for full details.

Previews (UP TO DATE):
[UPPER DECK](https://cdn.discordapp.com/attachments/546822932278673420/1130253978567970836/2023-07-16_17.45.02.png)
[LOWER DECK](https://cdn.discordapp.com/attachments/546822932278673420/1130253979486527578/2023-07-16_17.45.14.png)

Originals for comparison:
[UPPER DECK](https://cdn.discordapp.com/attachments/546822932278673420/1126678446421065839/2023-07-06_20.56.24.png)
[LOWER DECK](https://cdn.discordapp.com/attachments/546822932278673420/1126678445653512222/2023-07-06_20.56.14.png)

## Changelog

:cl:
add: [Biodome] Dirt to maintinence as a whole
add: [Biodome] More fuel and water tanks to maintinence
add: [Biodome] More fire and emergency closets to maintinence
add: [Biodome] More atmospheric machine spawns to maintinence
add: [Biodome] Blacksmith work area to Cargo
add: [Biodome] NT Representative/Consult Office just below Cargo in the upper main hall
add: [Biodome] Barber Shop/Salon to lower Service, where the Teleporter was
add: [Biodome] Directional indicator decals to the main hall leading to each department (or close enough to)
add: [Biodome] Directional indicator wall decals to the lower Service area pointing to each department (relatively)
add: [Biodome] The remainder of the Navigation Destinations (what leads you to departments when using the Navigation function)
add: [Biodome] Warning decals to all upper level stairways and elevators
add: [Biodome] An airlock to maintinence between Public Commisary and Engineering entryway
add: [Biodome] An external airlock set to Atmospherics maintinence for techs to repair the gas containers
add: [Biodome] Two cookies for the Mortician to replace the ones outside on the floor
add: [Biodome] Two hot cocoa for the Psychology room to better represent the theme
add: [Biodome] Secateurs to public garden and prison garden
add: [Biodome] Nspect objects added to Security
add: [Biodome] Keycard Authenticators added to CMO, Bridge, and Captain
add: [Biodome] Pipenets to maintinence to make it less visually bland
add: [Biodome] Wires to maintinence to make it less visually bland
add: [Biodome] Girder, grille, trash, dirt, etc spawners to maintinence to divvy up straight halls of nothing
add: [Biodome] Solar array construction crates added to lower starboard bow solar array and upper power bow solar array
add: [Biodome] More painting slots in the Library
add: [Biodome] Lights to various spots in the overarching Service area
add: [Biodome] Lightswitches in rooms that lacked them
add: [Biodome] Disposal bins in rooms that lacked them
del: [Biodome] Now redundant pipenet and wiring from maintinence changes
del: [Biodome] Two floor cookies outside of the Morgue
del: [Biodome] Removed Cargo elevator
del: [Biodome] Misplaced lightswitch at Chapel dumping area
del: [Biodome] Snowmen from outside of Medbay
del: [Biodome] Ghost-role MULE spawn in maintinence
del: [Biodome] Nearly all of the "planty" atmospheric holobarriers
del: [Biodome] Peeping tom hidden room connecting to Dorm 7
del: [Biodome] Directional windows and windoor from lower Chemistry area due to cramping
fix: [Biodome] Boat tram now properly increments in tram incidents panel
fix: [Biodome] Detective should now properly receive mail via disposals
fix: [Biodome] Chapel burial outlet should now properly be connected via disposals
fix: [Biodome] Atmospherics nitrogen gas tank sensor is now the correct one
fix: [Biodome] Lower starboard bow solars no longer have a looping powernet
fix: [Biodome] Lower electrical maintinence no longer has a looping powernet
fix: [Biodome] Connected Arrivals Restrooms plumbing to chemical synthesizer
fix: [Biodome] Geneticist shutters button now properly connected to shutters
fix: [Biodome] Aft Starboard solar array access fixed
fix: [Biodome] Medbay Orderly airlock now has the correct access
fix: [Biodome] Snow section outside of Cryogenics Bay no longer has airless snow tiles
fix: [Biodome] Navigator properly points to Research and Vault now
fix: [Biodome] AI Sat camera networks and motion sensing restored to parity
tweak: [Biodome] Various visual changes spanning the entire station to make it less eyesearing and/or more coherent in overall aesthetics (most notably the fake pits in the public area that contains Security and Engineering's entryway)
tweak: [Biodome] Various excessive decals across the entire station, either to be unified into a less "playplace" style or completely removed
tweak: [Biodome] Railings, floors, decals, and windows have been touched up across the entire station to create parity between Biodome and other in-rotation maps
tweak: [Biodome] Various maintinence rooms have been revamped entirely or touched up
tweak: [Biodome] Science break room is no longer a dump
tweak: [Biodome] Mining lobby has been expanded to have private rooms
tweak: [Biodome] Lengthened the Service transit tube and added an additional stop midway
tweak: [Biodome] Boat tram variables renamed to be more in line with other maps
tweak: [Biodome] Teleporter has been moved over to make room for the Barber/Salon
tweak: [Biodome] The crab habitat has been shrunk
tweak: [Biodome] Kitchen layout has been changed to be less CBT to access machines
tweak: [Biodome] Public Garden now has a predetermined set of produce instead of randomized seeds
tweak: [Biodome] Engineering's upper entryway is now more official looking to cause less confusion in locating it
tweak: [Biodome] Research's MULE delivery point has been moved from the EXPERIMENTOR corner to the main Research hall for clarity purposes
tweak: [Biodome] Replaced broken Boating Incidents sign with working Supermatter Delam counter Tram Incidents combination sign
tweak: [Biodome] Restructured the Bridge consoles so they were better grouped together
tweak: [Biodome] Widened the hallways into the Bridge from one wide to two wide
tweak: [Biodome] Captain's elevator has been replaced with a stairwell after reports of Secoffs falling down like lemmings
tweak: [Biodome] You can now visit the carp exhibit in northern upper Service
tweak: [Biodome] Recycler grinder has been moved over one to prevent trash from auto-grinding out of the chute
/:cl:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -243,3 +243,4 @@ define_sanity_output.txt
 # ezdb
 /db/
 /config/ezdb.txt
+tgstation.dme
```
Comment (nevimer) [inline .gitignore:246]: can't do this

Comment (TripleShades) [reply .gitignore:246]: Yeah that was my bad, clicked the wrong thing in my program

**Comment by TripleShades** [comment]
Did all the changes that I thought should be done/were egregiously in need of doing.
Writing changelog tomorrow.

**Comment by zeroisthebiggay** [comment]
i don't see how this makes the map any less of an unintuitive mess to play

**Comment by TripleShades** [comment]
> Post SDMM screenshots of the map alongside your changelog please. (top right cogwheel -> screenshot)

Thanks, was wondering where they moved it. Previews posted.

> i don't see how this makes the map any less of an unintuitive mess to play

I'm hoping that the directional floor lines and additional signage will help signifigantly.

**Comment by TripleShades** [comment]
Removing from draft.

**Comment by TripleShades** [comment]
I'll update the previews tomorrow but the big topology change is adding an office for NT Consult/Rep and the Barber/Salon.

**Comment by TripleShades** [comment]
Previews are now up to date. Solved issue with boat tram.

**Comment by projectkepler-ru** [comment]
will have to wait for @KathrinBailey  to take a look at this
@TripleShades  may take a bit, if it they still haven't got back to it... I'll take a loo myself

**Comment by TripleShades** [comment]
> will have to wait

That's fine, I need to update the previews because they are once again out of date as of today.

**Comment by TripleShades** [comment]
Previews are now up to date again.

**Comment by nevimer** [comment]
merging, for more updates make a new pr, I'll need to mapmerge it after this is done

**Comment by Profakos** [comment]
Thanks for alerting me to various errors, like the CMO lacking a keyauth console. Bridge and Captain already had one though, I can see two in your previous. Do those rooms have two of those on your maps?

I really like the xenobio park being moved to the Science break room, so I have done the same, but I am keeping at least one trash pile.

I also removed the "Peeping tom hidden room", realizing it was a bad place to put one. (Those secret rooms all over the map are rooms where the sociologist who built this station would teleport to, to observe the crew, had the escape shuttle not been called)

You probably have to replace the "planty" holobarriers with something else, they served the purpose of dividing up the air flow (and fires) of the big open dome to avoid lag while permitting movement for people who try to move between roofs. They were unfortunately  never updated to not have codersprites.

**Comment by TripleShades** [comment]
> Thanks for alerting me to various errors, like the CMO lacking a keyauth console. Bridge and Captain already had one though, I can see two in your previous. Do those rooms have two of those on your maps?

They do have two, I must have simply blinded them while being sleepy. Thank you for notifying me.

> I also removed the "Peeping tom hidden room", realizing it was a bad place to put one. (Those secret rooms all over the map are rooms where the sociologist who built this station would teleport to, to observe the crew, had the escape shuttle not been called)

Going to be real, none of the maint rooms really have cohesion to even support this idea. It wont matter in the future as I will be redoing the entirety of maints to coincide with redoing disposals.

> You probably have to replace the "planty" holobarriers with something else, they served the purpose of dividing up the air flow (and fires) of the big open dome to avoid lag while permitting movement for people who try to move between roofs. They were unfortunately never updated to not have codersprites.

I left in the holobarriers on the upper Service area windows, as I thought they made decent enough markers for the "bio" part of the dome, which is now entirely Service.
Still, I have been asked to add more greenery to the entire station, so that may change, and they will be removed at that time.

**Pull request merged**